packages/ckeditor5-list/src/listreversedcommand.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license Copyright (c) 2003-2021, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module list/listreversedcommand
+ */
+
+import { Command } from 'ckeditor5/src/core';
+import { getListItemsOfSelectedList } from './utils';
+
+/**
+ * The list reversed command. It is used by the {@link module:list/liststyle~ListStyle list style feature}.
```

Comment:
A few words about what it does would be nice. For instance, which model attribute is changed by this command.